test/parallel/test-tls-inception.js

```diff
@@ -59,11 +61,13 @@ a.listen(common.PORT, function() {
       rejectUnauthorized: false
     });
     ssl.setEncoding('utf8');
+    var buf = '';
     ssl.once('data', function(data) {
-      assert.equal('hello', data);
+      buf += data;
       gotHello = true;
     });
```

Comment:
`gotHello` seems to be necessary, but could be replaced with `common.mustCall` in `ssl.on('end', ...)`